### DISCUSSION TITLE
New game add? Life on Earth: reimagined

### DISCUSSION THREAD

**Discussion by djdaveet** [sanni/cartreader#885] [Need help]
So I picked up a copy of Life on Earth: Reimagined:

https://kai-magazine-software.fwscart.com/product/pre-order-(second-batch)--life-on-earth--reimagined-for-the-sega-genesis-(genesis-cover)

And would like to backup the ROM.

It appears the issue is that it's not in the database, and despite updating to the latest firmware and updating the files on the SD card, it fails the checksum.

Can someone help?

**Comment by PsyK0p4T** [comment]
If the game is quite new, and no dump reported yet, yeah the rom hashes and checksum are most probably unknown and not present in the MD.txt database
Feel free to share your results with Datomatic, or here, but not the rom file please.

**Comment by djdaveet** [comment]
As this is my first one and I’m pretty new at this, could I trouble you for a link to some guidance on “how to”?   

I’m not getting a rom — it errors out, so I just have the error files.   I’m happy to supply the results (not ROM) to help others.

**Comment by djdaveet** [comment]
And you will need the logs, so here:

OSCR HW3 V13.1

[+] Mega Drive/Genesis

[+] Game Cartridge


Cart Info
 
Name: SAMPLEPROGRAM
bramCheck: E100
Size: 8 MBit
ChkS: 6BD5
Sram: 256 KBit
 
Press Button...

[+] Read Rom


Saving to MD/ROM/SAMPLEPROGRAM/13/...
[*******************]
Internal checksum...Error
4FD3

CRC32... 423E89C8 -> Not found

**Reply by PsyK0p4T** [reply]
Hmm I guess you should have a ROM file in ".bin" format, placed in your MD/ROM/SAMPLEPROGRAM/13/... folder. 
Even if the game or crc are unknown, the file should be there. The internal checksum error you got here just appears because data doesn't match the value specified in the rom header.
Are you using discord ? try to reach me there, I can probably help on this.

**Reply by iamaldovanucci** [reply]
I have a copy of this game and tried dumping it as well. It only dumps 1024k and loads as some random garbace on screen. I'm guessing there is some sort of mapper being used but I'm not sure how to proceed.

**Reply by PsyK0p4T** [reply]
Are you using the very latest MD.ino file flashed on your reader ? The current one from dev branch should work.

**Reply by iamaldovanucci** [reply]
Just updated to the latest. It is read as SAMPLEPROGRAM and is 32MBitin size. The dumped file results in an on screen message about putting the original cartridge in an original console. Is there DRM on this cart?

**Reply by PsyK0p4T** [reply]
Yup, 32 Mbit / 4 MB was the expected size. 
If the file was named SAMPLEPROGRAM.bin, it could mean that you haven't updated the MD.txt file on your SD card, or that your file got a different CRC32 that previously known dump. Can you post its crc here please ?

**Reply by iamaldovanucci** [reply]
OSCR HW5 V13.2

[+] Mega Drive/Genesis

[+] Game Cartridge


Cart Info
 
Name: SAMPLEPROGRAM
bramCheck: E100
Size: 32 MBit
ChkS: 1FEA
Sram: 256 KBit
 
Press Button...

[+] Read ROM


Saving to MD/ROM/SAMPLEPROGRAM/85/...
[*******************]
Internal checksum...Error
34DE

CRC32... 3D38E3CF -> Not found

**Reply by iamaldovanucci** [reply]
Turns out this is expected there is a "patch" for this gam to run on flashcarts/emulators. What is the best way to share what I've found?

**Reply by PsyK0p4T** [reply]
Yeah a patch/hack is probably needed if the game got an AP check.

**Comment by djdaveet** [comment]
Sent you a Discord friend request.

**Comment by Anguka1** [comment]
Hello, has this been figured out? I would be willing to pay for the rom for this game. If the ones making it don’t want my money then someone else should have it.

**Reply by Anguka1** [reply]
To clarify I would like to use it on my iPhone, through delta so I can screen mirror it to my Apple TV box. That’s my setup I use my ps4 controller.

**Reply by Anguka1** [reply]
lol i made a mistake. I forgot delta doesn’t have genesis on the iPhone. I would be using RetroArch for any genesis games.

**Reply by Cephiros** [reply]
Please do not request roms, we do not support piracy here.

**Comment by Anguka1** [comment]
Im trying to purchase the rom. Im not sure if you understand but sega
genesis consoles have been out of print for decades.

**Reply by partlyhuman** [reply]
This project, the OSCR, is a device that allows you to take cartridges you physically own and extract the ROM. If you'd like to do that, you're in the right place. If you'd like to obtain the ROM any other way, I'm afraid this is the wrong place, please try elsewhere!

**Comment by Anguka1** [comment]
Sounds a bit difficult to figure out for me as I was never great with
computers. But I guess it’s my only option for now. Thank you for your
time.
